topics/introduction/metadata.yaml

```diff
@@ -248,6 +248,13 @@ material:
       - "Gain understanding on navigating and manipulating histories"
     time_estimation: "30 min"
     key_points:
+      - In Galaxy, data is stored in datasets and organized in histories.
+      - The right pane shows the current history. It displays the name of the history, the tags and annotations assigned to it, and the size of the data in it.
+      - The datasets are shown differently depending on their status: queued, running, ok, error, or paused.
+        - The summary view for each dataset can be expanded to show additional information.
```

Comment:
there are a few too many spaces on beginning of this line and next

Reply:
Thank you for pointing this out. I removed the extra spaces.